workflow-bot-app/src/main/java/com/symphony/bdk/workflow/api/v1/dto/ActivityView.java

```diff
@@ -0,0 +1,22 @@
+package com.symphony.bdk.workflow.api.v1.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Builder
+public class ActivityView {
+  private String workflowId;
+  private String instanceId;
+  private String activityId;
+  private String type; //TODO: enum needed here
+  private InstanceStatusEnum status;
```

Comment:
do we need the instance status inside an activity ?

Reply:
This is the activity instance status. An activity has instance like the process.
Since we always get this info from the historicService, it will always be "COMPLETED" so maybe we can get ride of it. Activity instances with ongoing execution should appear in the runtimeService, but they either stay there for a very short time or they are not even set there because the database is updated when no work is ongoing (maybe Camunda updates database in a batch or the process is synchronous..)

Reply:
statusEnum sounds good, avoid the confusion between process instance and activity instance